src/plugins/ovp/components/video-screen/index.js

```diff
@@ -36,6 +41,31 @@ function VideoScreen(props) {
   const canDelete = policy.isGranted(`${qname}:delete`);
   const canUpdate = policy.isGranted(`${qname}:update`);
 
+  const schema = node.schema();
+  const hasMedialiveChannel = schema.hasMixin('triniti:ovp.medialive:mixin:has-channel') && node.has('medialive_channel_arn');
+
+  const medialiveRequest = React.useMemo(() => {
+    if (!hasMedialiveChannel || !nodeRef) {
+      return null;
+    }
+    return GetNodeRequestV1.create()
+      .set('node_ref', NodeRef.fromString(`${nodeRef}`))
+      .addToSet('derefs', ['medialive_channel_state']);
+  }, [hasMedialiveChannel, nodeRef]);
```

Comment:
@meghedi this seems like it should be more of a `useMediaLive` or similar, but i guess since it's only in this component it's alright